LightBlue.Tests/Standalone/StandaloneAzureBlockBlobCopyByUriTests.cs

```diff
@@ -0,0 +1,172 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+using ExpectedObjects;
+using ExpectedObjects.Comparisons;
+
+using LightBlue.Standalone;
+
+using Microsoft.WindowsAzure.Storage.Blob;
+
+using Xunit;
+using Xunit.Extensions;
+
+namespace LightBlue.Tests.Standalone
+{
+    public class StandaloneAzureBlockBlobCopyByUriTests : StandaloneAzureTestsBase
+    {
+        public StandaloneAzureBlockBlobCopyByUriTests()
+            : base(DirectoryType.Container)
+        {}
+
+        public static IEnumerable<object[]> CopyBlobNames
+        {
+            get
+            {
+                yield return new object[] { "source", "destination" };
+                yield return new object[] { "source", @"destination\with\path" };
+                yield return new object[] { "source", "destination/with/path/alternate" };
+                yield return new object[] { @"source\with\path", "destination" };
+                yield return new object[] { "source/with/path/alternate", "destination" };
+                yield return new object[] { @"source\with\path", @"destination\with\path" };
+            }
+        }
+
+        [Theory]
+        [PropertyData("CopyBlobNames")]
+        public void CanCopyBlob(string source, string destination)
+        {
+            var buffer = Encoding.UTF8.GetBytes("File content");
+            var sourceBlob = new StandaloneAzureBlockBlob(BasePath, source);
+            sourceBlob.UploadFromByteArrayAsync(buffer, 0, buffer.Length).Wait();
+
+            var destinationBlob = new StandaloneAzureBlockBlob(BasePath, destination);
+            destinationBlob.StartCopyFromBlob(sourceBlob.Uri);
+
+            Assert.Equal("File content", File.ReadAllText(destinationBlob.Uri.LocalPath));
```

Comment:
Why not `Assert.Equal( File.ReadAllText(sourceBlob.Uri.LocalPath, File.ReadAllText(destinationBlob.Uri.LocalPath));`?